pkg/installation/install.go

```diff
@@ -64,33 +64,32 @@ func downloadAndMove(version, uri string, fos []index.FileOperation, downloadPat
 // to not get the plugin dir in a bad state if it fails during the process.
 func Install(p environment.Paths, plugin index.Plugin, forceDownloadFile string) error {
 	glog.V(2).Infof("Looking for installed versions")
-	_, ok, err := findInstalledPluginVersion(p.InstallPath(), p.BinPath(), plugin.Name)
-	if err != nil {
-		return err
-	}
-	if ok {
+	_, err := receipt.Load(p.PluginInstallReceiptPath(plugin.Name))
```

Comment:
As you are discarding the return value, could you also do a simpler `Lstat`?

Reply:
I think we need a `receipt.Exists`. Let's add that.

Reply:
On a second thought, I'm thinking if an empty file named plugin.yaml exists instead of the actual receipt, despite being harmless today, it might complicate things.

This doesn't have a lot of cost. Can we keep?